modules/gapi/CMakeLists.txt

```diff
@@ -138,5 +143,14 @@ if(TARGET opencv_test_gapi)
   target_link_libraries(opencv_test_gapi PRIVATE ade)
 endif()
 
+find_package(PlaidML2 CONFIG QUIET)
```

Comment:
Like many other optional dependencies of OpenCV this requires flag which allows explicit disabling of these feature.

- `WITH_PLAIDML`
- `OFF` by default (until there is documentation/tutorial and regular testing)

Reply:
Fixed